### DISCUSSION TITLE
Pull request table does not show reviewers

### DISCUSSION THREAD

**Discussion by andre-merzky** [community/community#37961] [Pull Requests]
I miss the option to quickly see what PRs need reviews by me (or a specific other developer).  At the moment I have to open each PR and manually check who is assigned as reviewer - the overview table does not provide that information.

How do others handle this?   It seems like such a small issue, but I stumble over it again and again...

Thanks, Andre.

**Comment by ossanna16** [comment]
Hi there @andre-merzky and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
